CMakeLists.txt

```diff
@@ -627,6 +639,7 @@ option(BUILD_PYTHON_BINDINGS "Build Python bindings" OFF)
 option(BUILD_CV_EXAMPLES "Build OpenCV examples" OFF)
 option(BUILD_PCL_EXAMPLES "Build PCL examples" OFF)
 option(BUILD_NODEJS_BINDINGS "Build Node.js bindings" OFF)
+option(DISABLE_EASYLOGGINGPP "Disable the inclusion of EasyLogging++ in the build." OFF)
```

Comment:
I think it would make more sense to use positive nouns instead of this double negation (and also follow the rest of the naming convention) i.e:
1. Use `BUILD_EASYLOGGINGPP ` with default to `ON` instead of the current way.
2. Change the rest of the changes follow - instead of removing files and flags, add them when building with easylogging